cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProfileHttpHandler.java

```diff
@@ -130,8 +131,9 @@ public void deleteSystemProfile(HttpRequest request, HttpResponder responder,
   @Path("/namespaces/{namespace-id}/profiles")
   public void getProfiles(HttpRequest request, HttpResponder responder,
                           @PathParam("namespace-id") String namespaceId,
-                          @QueryParam("includeSystem") @DefaultValue("false") String includeSystem) {
-    NamespaceId namespace = new NamespaceId(namespaceId);
+                          @QueryParam("includeSystem") @DefaultValue("false") String includeSystem)
```

Comment:
Should we still allow includeSystem as a query param here if we are restricting the use of namespaced api for profiles?

Reply:
we still need to support this to allow listing both user and system profiles in a namespace. This is why we did a top level API instead of keeping everything separate.